audit/format.go

```diff
@@ -25,14 +25,14 @@ type AuditFormatter struct {
 	AuditFormatWriter
 }
 
-func (f *AuditFormatter) FormatRequest(
-	w io.Writer,
-	config FormatterConfig,
-	auth *logical.Auth,
-	req *logical.Request,
-	inErr error) error {
-
-	if req == nil {
+var _ Formatter = (*AuditFormatter)(nil)
+
+func (f *AuditFormatter) FormatRequest(w io.Writer, config FormatterConfig, in *LogInput) error {
+	if in == nil {
+		in = &LogInput{}
+	}
+
+	if in.Request == nil {
```

Comment:
This and the above `if` can be combined into simply `if in == nil || in.Request == nil`.